numpy/random/generator.pyx

```diff
@@ -3475,21 +3492,41 @@ cdef class Generator:
 
         # GH10839, ensure double to make tol meaningful
         cov = cov.astype(np.double)
-        (u, s, v) = svd(cov)
+        if method == 'svd':
+            from numpy.dual import svd
+            (u, s, vh) = svd(cov)
+        elif method == 'eigh':
+            from numpy.dual import eigh
+            (s, u)  = eigh(cov)
```

Comment:
```suggestion
            # could call linalg.svd(hermitian=True), but that calculates a vh we don't need
            (s, u)  = eigh(cov)
```

Reply:
It is derived from u, so only memory cost for extra array.

Reply:
Plus the cost of the multiplication - not high, but clearly high enough that you decided not to use it. Thanks for the comment.